raster.py

```diff
@@ -223,8 +223,9 @@ def save_numpy_to_geotiff(bounds: Bounds, prediction: np.ma.MaskedArray, path: s
 
 def coords_to_indices(bounds: Bounds, x: float, y: float):
   if x < bounds.minx or x > bounds.maxx or y < bounds.miny or y > bounds.maxy:
-    raise ValueError("Coordinates out of bounds")
-
+    #raise ValueError("Coordinates out of bounds")
```

Comment:
Why do we want to do this? Do we not want to make our program raise this anomaly?

Reply:
It was a change made by Ruben in order to generate the random coordinates. If we keep as is, we got "coordinates out of bounds" all the time =/